applications/services/desktop/scenes/desktop_scene_pin_input.c

```diff
@@ -127,12 +128,12 @@ bool desktop_scene_pin_input_on_event(void* context, SceneManagerEvent event) {
         case DesktopPinInputEventResetWrongPinLabel:
             desktop_scene_locked_light_red(false);
             desktop_view_pin_input_set_label_primary(desktop->pin_input_view, 0, 0, NULL);
-            FuriString* enter_pin_string = furi_string_alloc();
-            furi_string_set_str(enter_pin_string, "Enter PIN (Wrong Attempts: ");
+            state->enter_pin_string = furi_string_alloc();
```

Comment:
Remove this line, `on_enter` has already allocated the string. You only need to modify it, not allocate again.

Reply:
couldn´t just leave it away, did the things below, but thanks for pointing out